settings.gradle

```diff
@@ -167,3 +167,6 @@ include "beam-test-infra-metrics"
 project(":beam-test-infra-metrics").dir = file(".test-infra/metrics")
 include "beam-test-tools"
 project(":beam-test-tools").dir = file(".test-infra/tools")
+include 'sdks:java:extensions:ml'
+findProject(':sdks:java:extensions:ml')?.name = 'ml'
```

Comment:
I don't see names being set elsewhere, why is this project different?

Reply:
That's an oversight on my part, I'll delete this.
The inclusion was already added in earlier PRs, the rebase will make this change unnecessary.